server/routes/users.js

```diff
@@ -145,4 +145,59 @@ router.post('/remove-following', jwt.verifyJWT, async (req, res, next) => {
   }
 })
 
-module.exports = router
\ No newline at end of file
+router.post('/add-follower-favorite', jwt.verifyJWT, async (req, res, next) => {
```

Comment:
Where is this getting called? I also don't think this should be part of this PR.

Reply:
Yea, idk how this got in here.